js/rtl.js

```diff
@@ -1,50 +0,0 @@
-/*
- * dmartl.js
- * DM Auto RTL - Auto RTL all inline style in page using jquery
- * Autor: Danoosh Miralayi
- * Website: presta-shop.ir
- * License: MIT
- * Find it here: https://github.com/Danoosh/DM-Auto-RTL
```

Comment:
Maybe we can ask @Danoosh if he remembers why this file was introduced, and tell us if it is still relevant ? :)

Reply:
I think it was useful for early versions of 1.6.

Reply:
In current PS, I couldn't find any inline styles that requires compatibility for RTL languages (e.g. text-align, float or background-position).